from gtmcore.exceptions.exceptions import GigantumException, GigantumLockedException
